doc/source/whatsnew/v0.19.0.txt

```diff
@@ -864,3 +865,4 @@ Bug Fixes
 - Bug in ``.to_excel()`` when DataFrame contains a MultiIndex which contains a label with a NaN value (:issue:`13511`)
 - Bug in ``pd.read_csv`` in Python 2.x with non-UTF8 encoded, multi-character separated data (:issue:`3404`)
 - Bug in ``Index`` raises ``KeyError`` displaying incorrect column when column is not in the df and columns contains duplicate values (:issue:`13822`)
+- Bug in ``Period`` and ``PeriodIndex`` creating wrong dates when frequency has multiple offsets (:issue:`13874`)
```

Comment:
"combined offsets" or "combined aliases" should be better. 

- http://pandas.pydata.org/pandas-docs/stable/timeseries.html#combining-aliases